modules/core/src/main/scala/itchart/scala/core/BikeTrevelTime.scala

```diff
@@ -0,0 +1,6 @@
+package itchart.scala.core
+
+import java.util.Date
+
+final case class BikeTravelTime(startTime:Date,
```

Comment:
It's possible to define all you model classes in one file - it will be much more cleaner.
Also, BikeTravelTime cold be defined as one-liner